src/pages/resultsView/enrichments/MiniScatterChart.tsx

```diff
@@ -80,9 +80,12 @@ export default class MiniScatterChart extends React.Component<IMiniScatterChartP
                                 tickLabels: { padding: 135, fill: "black" }, axisLabel: { padding: 165, fill: "black", fontSize: 16 },
                                 axis: { stroke: "black", strokeWidth: 1 }, grid: { stroke: "#eeeeee", strokeDasharray: "none" }, ticks: { size: 0 }
                             }} />
-                        <VictoryLabel text={"← " + this.props.xAxisLeftLabel} x={60} y={300} style={{ fill: "blue" }} />
-                        <VictoryLabel text={this.props.xAxisRightLabel + " →"} x={200} y={300} style={{ fill: "red" }} />
-                        <VictoryLabel text="significance →" x={320} y={210} angle={-90} style={{ fontSize: 16 }} />
+                        <VictoryLabel text={"← " + this.props.xAxisLeftLabel} x={60} y={300} 
+                            style={{ fill: "blue", fontSize: 12, fontFamily: "Arial, Helvetica" }} />
```

Comment:
how come we're specifying font here?  should be inherited from theme

Reply:
for some reason, standalone `VictoryLabel`s don't inherit from the theme. I tried adding a `label` section to the `CBIOPORTAL_VICTORY_THEME` object but didn't work.